test/fixtures/dialects/bigquery/export_statement.yml

```diff
@@ -0,0 +1,298 @@
+# YML test files are auto-generated from SQL files and should not be edited by
+# hand. To help enforce this, the "hash" field in the file must match a hash
+# computed by SQLFluff when running the tests. Please run
+# `python test/generate_parse_fixture_yml.py`  to generate them after adding or
+# altering SQL files.
+_hash: 60fc6c7bce3697efc8ae912ed01175617199d44264c863a6b00becf23bc704f6
+file:
+- statement:
+    export_statement:
+    - keyword: EXPORT
+    - keyword: DATA
+    - keyword: WITH
+    - keyword: CONNECTION
+    - object_reference:
+      - identifier: PROJECT_ID
+      - dot: .
+      - identifier: LOCATION
+      - dot: .
+      - identifier: CONNECTION_ID
+    - keyword: OPTIONS
+    - bracketed:
+      - start_bracket: (
+      - raw: uri
+      - comparison_operator:
+          raw_comparison_operator: '='
+      - literal: "'gs://bucket/folder/*.csv'"
+      - comma: ','
+      - raw: format
+      - comparison_operator:
+          raw_comparison_operator: '='
+      - literal: "'CSV'"
+      - comma: ','
+      - raw: overwrite
+      - comparison_operator:
+          raw_comparison_operator: '='
+      - keyword: 'true'
+      - comma: ','
+      - raw: header
+      - comparison_operator:
+          raw_comparison_operator: '='
+      - keyword: 'true'
+      - comma: ','
+      - raw: field_delimiter
+      - comparison_operator:
+          raw_comparison_operator: '='
+      - literal: "';'"
+      - end_bracket: )
+    - keyword: AS
+    - select_statement:
+        select_clause:
+        - keyword: SELECT
+        - select_clause_element:
+            column_reference:
+              identifier: field1
+        - comma: ','
+        - select_clause_element:
+            column_reference:
+              identifier: field2
+        from_clause:
+          keyword: FROM
+          from_expression:
+            from_expression_element:
+              table_expression:
+                table_reference:
+                - identifier: mydataset
+                - dot: .
+                - identifier: table1
+        orderby_clause:
+        - keyword: ORDER
+        - keyword: BY
+        - column_reference:
+            identifier: field1
+- statement_terminator: ;
+- statement:
+    export_statement:
+    - keyword: EXPORT
+    - keyword: DATA
+    - keyword: WITH
+    - keyword: CONNECTION
+    - object_reference:
+        identifier: '`PROJECT_ID.LOCATION.CONNECTION_ID`'
+    - keyword: OPTIONS
+    - bracketed:
+      - start_bracket: (
+      - raw: uri
```

Comment:
do we want to give this a type?

Reply:
Good spot - done in https://github.com/sqlfluff/sqlfluff/pull/3177/commits/15145a2b69dbb96e0161fc7cae360ed0059f5e66